phpunit-api-coverage.xml

```diff
@@ -13,4 +11,8 @@
             <directory suffix=".php">src</directory>
         </whitelist>
     </filter>
+
+    <php>
+        <const name="SDK_TESTING_ENV" value="api_coverage" />
```

Comment:
The former, using `<env>`.

My logic would be that environment variables were designed for these things. Having phpunit generate constants trips up static analysis, for example PhpStorm will claim that `SDK_TESTING_ENV` is an undefined constant.